examples/with-thirdweb-typescript/LICENSE.md

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Our examples do not have a LICENSE, I'm curious why this was added here?

Reply:
Oh my bad. I will remove the licenses